README.md

```diff
@@ -272,6 +273,12 @@ Alternatively you can pull from [Docker Hub](https://hub.docker.com/r/kuchy/pack
 docker pull kuchy/packtpub-crawler
 ```
 
+### cronjob
+Add this to your crontab to run the job daily at 9:
+```
+00 09 * * * /usr/bin/python PATH_TO_CRAWLER/script/spider.py --config PATH_TO_CRAWLER/config/prod.cfg --all --extras --notify join >> /tmp/packtpub.log 2>&1
+```
+
 ### Development (only for spidering)
```

Comment:
This should close the issue #8 
I would add something like: add the job with `crontab -e`.

I just tried now on the fly the command with `*/2 * * * *`, but I'm missing something cos doesn't work, did you edit any permission `chmod`? I've omitted the append and checked in the logs `grep CRON /var/log/syslog` but everything seems correct, but the book is not downloaded...